openwisp-config/files/openwisp.agent

```diff
@@ -562,12 +563,34 @@ update_info() {
 	rm $UPDATE_INFO
 }
 
+collect_error_logs() {
+	local last_update_line_no
+	# Save openwisp_config related logs to a temporary file
+	logread -e openwisp >"$TEMP_ERROR_LOG"
+	# Get the line number of the last configuration update
+	last_update_line_no=$(
+		grep "Local configuration outdated" -n \
+			"$TEMP_ERROR_LOG" \
+			| tail -1 \
+			| cut -d : -f 1
+	)
```

Comment:
Let's make sure that we can limit the maximum amount of lines which we can collect.
We want to avoid a potential situation in which for some nasty bug the collected log lines are huge and don't fit in the max post size. I think it would be better to give priority to the first lines in case we need to truncate, because if there's many log lines it's gonna be most probably some loop or something that keeps repeating itself over and over.